spec/player_spec.rb

```diff
@@ -0,0 +1,18 @@
+# rubocop: disable Lint/ParenthesesAsGroupedExpression
+
+require_relative '../lib/player'
+
+describe Player do
+  let (:player) { Player.new('gio', 'X') }
+
+  context 'initialize' do
+    it 'Take player name' do
+      expect(player.name).to eq('gio')
+    end
+    it 'Set player token' do
+      expect(player.token).to eq('X')
+    end
```

Comment:
## Negative scenario

- Specs cover positive and negative scenarios  (screenshot  from  Pathwright)
Example: there is a spec for correct usage of the method [e.g. it "creates object if all required attributes are passed"]
and also there is spec for the wrong usage of a method  [e.g. it "does not create the object if required attributes are missing"].

  - [ ] Thus, please add negative scenarios

Example:

```ruby
#positive scenario
 it 'Set player token' do
#negative scenario
 it 'doesn\'t Set player token ' do
```

Reply:
Hi, @bertil291utn, this method is not going to be false because we declare the  'X' for the first player at the beginning of the game.